build/run-tests.sh

```diff
@@ -1,25 +1,40 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 set -e
 
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
 function stop_containers() {
   docker-compose down
 }
 
 trap stop_containers EXIT
 
-if [[ ! -d ../verify-event-system-database-scripts ]]; then
+if [[ -z "${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}" ]]; then
+    export VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION=${SCRIPT_DIR}/../../verify-event-system-database-scripts
+fi
+
+echo Database Scripts are located at: ${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}
+
+if [[ ! -d "${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}" ]]; then
     echo "Cloning database migration scripts repo..."
-    pushd ..
+    pushd ${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}/..
```

Comment:
Will this work? If `${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}` doesn't exist, I wouldn't expect to be able to `cd` to it, and then go up a directory... My trivial attempts in other circumstances don't let me...

Reply:
Agree, I think you need `pushd $( dirname  ${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION} )` or something like that

Reply:
Good point ... well made ... it worked before last change, but you're quite right won't work now!

Reply:
Fix pushed